src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/CollectionInstrumentSvcClient.java

```diff
@@ -14,8 +45,40 @@
    * @return list of collection instruments matching the search string
    * @throws RestClientException something went wrong making http call
    */
-  List<CollectionInstrumentDTO> requestCollectionInstruments(String searchString)
-      throws RestClientException;
+  @Retryable(
```

Comment:
The `@Retryable` annotation should be used very sparingly. We can't have users waiting 25 seconds for a response.